JavaScriptSPA/authConfig.js

```diff
@@ -0,0 +1,19 @@
+ 
+  // Config object to be passed to Msal on creation
+  const msalConfig = {
+    auth: {
+      clientId: "Enter_the_Application_Id_Here",
+      authority: "Enter_the_Cloud_Instance_Id_Here/Enter_the_Tenant_Info_Here",
```

Comment:
The Azure portal replaces the endpoint host URL with protocol+domain+trailing slash. Update this to "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here"

Reply:
`Enter_the_Cloud_Instance_Id_Here` is not a supported token. The replacement token for this is `Enter_the_Authority_Endpoint_Host_Here`, so the whole authority is `"Enter_the_Authority_Endpoint_Host_HereEnter_the_Tenant_Info_Here"`.

Reply:
Please disregard